examples/SimpleRagas/assets/example_setupcode.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+import os
+import re
+import tempfile
+import lancedb
+
+import braintrust
+import markdownify
+import openai
+import requests
+
+# We're only going to embed a small subset of our total data for this example, but you can increase this number to embed more sections!
+NUM_SECTIONS = 20
+EMBEDDING_MODEL = "text-embedding-3-small"
+
+
+coda_qa_content_data = requests.get(
+    "https://gist.githubusercontent.com/wong-codaio/b8ea0e087f800971ca5ec9eef617273e/raw/39f8bd2ebdecee485021e20f2c1d40fd649a4c77/articles.json"
+).json()
+
+markdown_sections = [
```

Comment:
Another option could be to upload the cleaned sections, so that we could completely remove the `markdown_sections = [` list comprehension